pytest issue

=========================== short test summary info ============================
FAILED tests/llm/test_dr_write.py::test_write_composes_report - ValueError: Query must be at least 3 characters long
FAILED tests/llm/test_dr_write.py::test_run_pipeline - ValueError: Query must be at least 3 characters long
======= 2 failed, 10 passed, 1 skipped, 37 warnings in 78.52s (0:01:18) ========
Error: Process completed with exit code 1.

--- a/tests/llm/test_dr_write.py
+++ b/tests/llm/test_dr_write.py
@@ -13,7 +13,7 @@ def load_components():
         def validate_query(q):
             return True, ""
     oq.QueryManager = QM
-    sys.modules.setdefault("OvIntelligence.query_manager", oq)
+    sys.modules["OvIntelligence.query_manager"] = oq
     mf = types.ModuleType("omicverse.llm.model_factory")
     class DummyFactory:
         @staticmethod

